samples/Samples/Pages/OAuthExample.cshtml

```diff
@@ -0,0 +1,81 @@
+﻿@page
+@model Samples.Pages.OAuthExampleModel
+@{
+}
+
+<h1>OAuth with ASP.NET Core and Amazon Cognito</h1>
+
+<p>
+    Cognito support OAuth 2.0 authorization framework to make authenticated REST API calls. To identify the authenticated user to an
+    API a JSON Web Token (JWT) is generated by the OAuth provider, in this case Cognito, and passed as the <b>Authorization</b> header
+    to the REST API. The format of the value for Authorization header is "<i>bearer &lt;jwt-token&gt;</i>".
+</p>
+
+<h2>Retrieving current user's JWT token.</h2>
+
+<p>
+    In an ASP.NET Core application a JWT token can be retrieved for the current logged in user by injecting ASP.NET Core's UserManager&lt;CognitoUser&gt; service.
```

Comment:
Bold the class names (UserManager<CognitoUser>, UserManager, CognitoUser) like they are below?

Reply:
Done